client/gui/ShortcutHandler.h

```diff
@@ -0,0 +1,24 @@
+/*
+ * ShortcutHandler.h, part of VCMI engine
+ *
+ * Authors: listed in file AUTHORS in main folder
+ *
+ * License: GNU General Public License v2.0 or later
+ * Full text of license available in license.txt file, in main folder
+ *
+ */
+
+#pragma once
+
+enum class EShortcut;
+using SDL_Keycode = int32_t;
+
+class ShortcutHandler
+{
+public:
+	/// returns list of shortcuts assigned to provided SDL keycode
+	std::vector<EShortcut> translateKeycode(SDL_Keycode key);
+
+	/// attempts to find shortcut by its unique identifier. Returns EShortcut::NONE on failure
+	EShortcut findShortcut(const std::string & identifier );
```

Comment:
both could be either static or const

Reply:
True, however right now this is basically placeholder class for future implementation (hopefully for 1.3, unless I end up too busy).
Proper implementation won't be stateless so static is a no. Const makes sense.